includes/class-wc-payments-onboarding-service.php

```diff
@@ -655,7 +655,7 @@ public static function get_from( ?string $referer = null, ?array $get_params = n
 		if ( false !== strpos( $referer, 'path=/payments/onboarding' ) ) {
 			return self::FROM_ONBOARDING_WIZARD;
 		}
-		if ( false !== strpos( $referer, 'path=/payments/deposits' ) ) {
+		if ( false !== strpos( $referer, 'path=/payments/payouts' ) ) {
```

Comment:
I don't fully understand what this is for but is it possible that the value of `$referer` is `path=/payments/deposits`?

If so, we might want to do an `or` here.

```suggestion
		if ( false !== strpos( $referer, 'path=/payments/deposits' ) || false !== strpos( $referer, 'path=/payments/payouts' ) ) {
```